### ISSUE TITLE
Personas in interactive mode

### ISSUE THREAD

**Issue by christqoh** [facebookresearch/ParlAI#937]
partially continuing from #932 and #933 
When interacting with a ConvAI2 profilememory model through `interactive.py`, are replies from the model influenced by attention wrt to a persona?

**Comment by alexholdenmiller** [comment]
Nope, you have to provide one. For example you can say "your persona: you like pizza\n what do you think about pizza?"
